sdk/core/core-rest-pipeline/src/util/userAgentPlatform-browser.mts

```diff
@@ -31,24 +31,13 @@ interface NavigatorEx extends Navigator {
 function getBrandVersionString(
   brands: { brand: string; version: string }[],
 ): { brand: string; version: string } | undefined {
-  // Check for Microsoft Edge
-  const edge = brands.find((b) => b.brand === "Microsoft Edge");
-  if (edge) {
-    return edge;
-  }
-
-  // Check for Chrome
-  const chrome = brands.find((b) => b.brand === "Google Chrome");
-  if (chrome) {
-    return chrome;
-  }
-
-  // Check for Chromium
-  const chromium = brands.find((b) => b.brand === "Chromium");
-  if (chromium) {
-    return chromium;
+  const brandOrder = ["Google Chrome", "Microsoft Edge", "Brave", "Chromium"];
```

Comment:
curious: what else are in bands array? Is it useful to have all the info instead? or can we return first/last brand if they are sorted in some order?

Reply:
It should be sorted as it falls back to Chromium if not Brave, Google Chrome nor Microsoft Edge.

Reply:
I meant **if** `bands` are known to be sorted from generic to most specific, we could've returned the last entry of `brands`.